core/src/main/java/com/netflix/conductor/core/execution/ParametersUtils.java

```diff
@@ -214,7 +213,11 @@ private Object replaceVariables(String paramString, DocumentContext documentCont
                     }
 
                 } else {
-                    convertedValues[i] = documentContext.read(paramPath);
+                    try {
+                        convertedValues[i] = documentContext.read(paramPath);
+                    } catch (Exception e) {
```

Comment:
Add warning level logging

Reply:
thank you review, please review the https://github.com/Netflix/conductor/pull/988 , if it ok ,current pr will close